MetaMorpheus/TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -501,16 +501,32 @@ private void QuantificationAnalysis()
                 }
             }
 
+            // Determine if only select, high confidence peptides will be used for quant
+            List<string> peptideSequencesForQuantification = null;
+            if (Parameters.SearchParameters.SilacLabels == null && Parameters.SearchParameters.OnlyQuantifyConfidentPeptides)
+            {
+                peptideSequencesForQuantification = FilteredPsms.Filter(Parameters.AllPsms,
+                                CommonParameters,
+                                includeDecoys: false,
+                                includeContaminants: true,
+                                includeAmbiguous: false,
+                                includeAmbiguousMods: false,
+                                includeHighQValuePsms: false,
+                                filterAtPeptideLevel: true)
+                    .Select(pep => pep.FullSequence).Distinct().ToList();
```

Comment:
We are excluding ambiguous now, has that always been the case?

Reply:
yes